frontend/src/components/header/Header.tsx

```diff
@@ -1,17 +1,18 @@
 import React, {useEffect, useState} from 'react';
 import {NavigateFunction, useNavigate} from "react-router-dom";
 import {HeaderProps} from "./props";
-import {IRoute, mainRoutes, Paths} from "../../routes/routesConfig";
+import {IRoute, mainRoutes} from "../../routes/routesConfig";
 import {AppBar, Container, StyledEngineProvider, Toolbar} from '@mui/material';
-import DesktopLogo from "./DesktopLogo/DesktopLogo";
-import MobileMenu from "./MobileMenu/MobileMenu";
-import MobileLogo from "./MobileLogo/MobileLogo";
-import DesktopMenu from "./DesktopMenu/DesktopMenu";
-import UserMenu from "./UserMenu/UserMenu";
+import DesktopLogo from "./desktopLogo/DesktopLogo";
+import MobileMenu from "./mobileMenu/MobileMenu";
+import MobileLogo from "./mobileLogo/MobileLogo";
+import DesktopMenu from "./desktopMenu/DesktopMenu";
+import UserMenu from "./userMenu/UserMenu";
 import './styles.css';
+import {Headers, Paths} from "../../constansts/app";
 
-const pages = ['Home', 'TopUp', 'Transactions'];
-const settings = ['Profile', 'Logout'];
+const pages = [Headers.HOME, Headers.TOP_UP, Headers.TRANSACTIONS];
```

Comment:
move it to constants. It's not convenient to keep here these variables and then pass it to sub components. You can directly import them